synthetic_data_generation.py

```diff
@@ -111,12 +113,41 @@ def noise_generator(x):
         axs[i].axvline(x=true_freq, color='k', linestyle='--', alpha=0.4) #plotting vertical lines at the true frequencies            
 
 fig.tight_layout()
+
+
 # %%
 # now that we better understand how to use frequency estimation tools
 
 # lets try to simulate a signal with powerline noise.
 
 # create a signal that has powerline noise at 60Hz and at 6 harmonics (120, 180, 240, ...)
 # plot 1: plot the signal
+t, sinewave = sine_wave_generator(DURATION_S, SAMPLE_RATE, FREQUENCY, n_harmonics=N_HAR) 
+FREQUENCY = 60
```

Comment:
If `sine_wave_generator` is using these parameters as inputs to the function, is it actually using these values to evaluate the above line? Where should this move to ensure `sine_wave_generator` will use these as inputs?

Reply:
Oh I see, I have to move the parameters before the function

Reply:
exactly 👍🏾